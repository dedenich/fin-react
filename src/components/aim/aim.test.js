import React from 'react';
import renderer from 'react-test-renderer';

import Aim from './aim.jsx';

const aimMock = {
  amount: 1212,
  description: `Билет в Зеленоград`
};

it(`Aim renders correctly`, () => {
  const {amount, description} = aimMock;
  const tree = renderer
  .create(
      <Aim
        amount={amount}
        description={description}
      />
  )
  .toJSON();
  expect(tree).toMatchSnapshot();
});
