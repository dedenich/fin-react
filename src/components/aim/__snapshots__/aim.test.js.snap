// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Aim renders correctly 1`] = `
<div
  className="card aim-container"
  data-state="not empty"
  id="aim"
>
  <div>
    <div
      className="aim-container__curr-cont"
    >
      <h2>
        1212.00
      </h2>
      <div
        className="aim-container__cur-controls"
      >
        <div
          className="aim-container__controls_cur"
        >
          BYN
        </div>
      </div>
    </div>
    <div
      className="aim-container__aim-descr"
    >
      Билет в Зеленоград
    </div>
  </div>
</div>
`;
